pkg/asset/ignition/machine/master.go

```diff
@@ -63,3 +64,22 @@ func (a *Master) Name() string {
 func (a *Master) Files() []*asset.File {
 	return a.FileList
 }
+
+// Load returns the master ignitions from disk.
+func (a *Master) Load(f asset.FileFetcher) (found bool, err error) {
+	a.FileList, err = f.FetchByPattern(regexp.MustCompile(`^(master-(0|([1-9]\d*))\.ign)$`))
+	if len(a.FileList) == 0 {
+		return false, nil
+	}
+
+	configs := make([]igntypes.Config, len(a.FileList))
+	a.Configs = make([]*igntypes.Config, len(a.FileList))
+	for i, f := range a.FileList {
+		if err := json.Unmarshal(f.Data, &configs[i]); err != nil {
```

Comment:
Can we do the following instead of having the `configs` slice?

```
a.Configs[i] = &igntypes.Config{}
if err := json.Unmarshal(f.Data, a.Configs[i]); err != nil {
...
```